ministryofjustice/laa-apply-for-criminal-legal-aid
# CRIMAP-47 Redirect non passported

## Description of change

Adds DWP passporting check journey

## Link to relevant ticket

https://dsdmoj.atlassian.net/browse/CRIMAP-47
https://dsdmoj.atlassian.net/browse/CRIMAP-153

## Notes for reviewer

## Screenshots of changes (if applicable)
![0 0 0 0_3000_applications_47e125c4-9a1c-4542-9298-f9cad2769807_steps_client_benefit_check_result](https://user-images.githubusercontent.com/367349/196734885-4098dcfb-478c-48d7-8eaa-720dbd0e192d.png)

![0 0 0 0_3000_applications_6cbe1bc6-381e-465c-b458-5f46d7fe0649_steps_dwp_confirm_result](https://user-images.githubusercontent.com/367349/197773459-2b970d8b-19af-45a3-aeed-b7a91b9a65fe.png)


![0 0 0 0_3000_applications_47e125c4-9a1c-4542-9298-f9cad2769807_steps_client_confirm_details](https://user-images.githubusercontent.com/367349/196735082-9ba7cdf9-6836-4719-b3a4-b34482cdf58b.png)


### Before changes:

The DWP check just provided a yes or no

### After changes:

The journey now has the ability to agree or disagree with the result, and to confirm if client details are correct

## How to manually test the feature

After the National Insurance number page, you should be presented with the results of the DWP check and the onward journey. For NINOs that provide a confirmation of passporting benefits, see here https://dsdmoj.atlassian.net/wiki/spaces/CRIMAP/pages/4186800190/List+of+known+NINOs+for+the+MockBenefitCheckService

Otherwise, enter any valid NINO and follow the other path
